src/rgw/rgw_common.cc

```diff
@@ -1122,16 +1126,17 @@ bool verify_user_permission(const DoutPrefixProvider* dpp,
   if (usr_policy_res == Effect::Pass) {
     return false;
   }
-  else if (usr_policy_res == Effect::Allow) {
-    return true;
-  }
+
```

Comment:
the `usr_policy_res == Effect::Pass` block above can be removed as well, since both cases now `return false`

Reply:
Ok,will check this.